Tests/ApodiniTests/EndpointsTreeTests.swift

```diff
@@ -48,7 +48,7 @@ final class EndpointsTreeTests: ApodiniTests {
             // swiftlint:disable:next force_unwrapping
             (0...times!)
                 .map { _ in
-                    "Hello \(name) born in \(birthdate.year)!"
+                    "Hello \(name) born in \("birthdate.year")!"
```

Comment:
Those parentheses seem a bit odd to me? Is this intentional? Doesn't seem to impact anything but still.

Reply:
No, it isn't...I had to "uncomment" quite some code in order to find out why the compiler crashed and didn't catch this one when adding stuff back in. Thanks for the hint!